src/core/document.js

```diff
@@ -37,14 +37,15 @@ var Page = (function PageClosure() {
   }
 
   function Page(pdfManager, xref, pageIndex, pageDict, ref, fontCache,
```

Comment:
can we change `classFactory` to `pdfFunctionFactory` and also pass parameters as an object